app/jobs/dttp/check_consistency_job.rb

```diff
@@ -10,7 +10,7 @@ def perform(consistency_check_id)
       @dttp_placement_assignment_updated_date = Dttp::PlacementAssignments::Fetch.call(placement_assignment_dttp_id: trainee.placement_assignment_dttp_id).updated_at
 
       if contact_conflict || placement_assignment_conflict
-        SlackNotifierService.call(channel: Settings.slack.default_channel, message: "There is contact and/or Placement Assignment Conflict on Dttp", username: "DTTP Conflict Error")
+        SlackNotifierService.call(channel: Settings.slack.default_channel, message: "There is Contact and/or Placement Assignment conflict on Dttp", username: "DTTP Conflict Error")
```

Comment:
We should include a link to the trainee or at least the trainee ID in the message so as we know where to look. We could probably shorten it too... what about "Trainee {id} has been updated in DTTP"

Reply:
Will shorten and make message more descriptive, and itll also be a URL